plugins/rest/komand_rest/util/util.py

```diff
@@ -61,6 +63,32 @@ def first(sequence, default=""):
     return next((x for x in sequence if x), default)
 
 
+def check_headers_for_urlencoded(headers: Dict[str, str]) -> bool:
+    """
+    This method will check the headers for 'content-type' == 'application/x-www-form-urlencoded'
+    :param headers: Headers dict to read
+    :return: Boolean value indicating if the conditional is present
+    """
+    try:
+        for key, value in headers.items():
+            if key.lower() == "content-type" and value.lower() == "application/x-www-form-urlencoded":
+                return True
+    except AttributeError:
+        raise AttributeError("No headers found")
```

Comment:
I would delete that `try except` here because you have `params.get(Input.HEADER, {})` so it is always a dict object.